src/devices/sound/rf5c400.cpp

```diff
@@ -13,10 +13,12 @@
 namespace {
 
 int volume_table[256];
-double pan_table[0x64];
+double pan_table[256];
```

Comment:
The actual value being used for "pan" has a range of 0x00 to 0xff so I bumped the table size up accordingly. I am not sure if the actual pan values being calculated are correct but I don't have a way to do solid testing that. When beatmania III sets the "pan" of the normal channels to 0xe0 as described in the comments, it's still possible to very faintly hear the clean audio.